HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -18,6 +17,16 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+# Matching ECAL selective readout in particle flow, need a new input with online Selective Readout Flags
+def customiseFor17794(process):
+     for edproducer in producers_by_type(process, "PFRecHitProducer"):
+          if hasattr(edproducer,'producers'):
+               for pset in edproducer.producers:
+                    if (pset.name == 'PFEBRecHitCreator' or pset.name == 'PFEERecHitCreator'):
+                         if not hasattr(pset,'srFlags'):
+                              pset.srFlags = cms.InputTag('hltEcalDigis')
```

Comment:
this was earlier requested to be made available via fillDescriptions (a comment from @davidlange6 ).
Similar to a few other places, current fillDescriptions interface does not provide support for configurations of plugins.
(existsAs method would not clearly help in this case either because hltEcalDigis is not an obvious default).

So, this customise will have to stay here